### PR TITLE
checks fun

### PR DESCRIPTION
**Pull request by kevmoo**
- Cleanup mono_repo yaml files
- Update to latest mono_repo
- check: 100% code coverage of core extensions

### CODE REVIEW

**Comment by kevmoo** [comment]
@natebosch – you'll probably say you want separate PRs here...

File: pkgs/checks/test/extensions/core_test.dart [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+  return rejection;
+}
+
+void main() {
+  group('TypeChecks', () {
+    test('isA', () {
+      checkThat(1).isA<int>();
+
+      expect(
```
Comment (natebosch) [inline pkgs/checks/test/extensions/core_test.dart:18]: We should not use `expect` or `Matcher` in these tests.

Comment (kevmoo) [reply pkgs/checks/test/extensions/core_test.dart:18]: To bootstrap it seemed a good idea – so we're not leaning on the thing we're testing.

Comment (natebosch) [reply pkgs/checks/test/extensions/core_test.dart:18]: I disagree, that sounds like building a bunch of debt.

We had no problem using `expect` to test `expect`. I don't see what we'll have any difficulty here.

Comment (kevmoo) [reply pkgs/checks/test/extensions/core_test.dart]: fixed!

File: pkgs/checks/test/extensions/core_test.dart [hunk 2]
```diff
@@ -0,0 +1,159 @@
+import 'package:checks/checks.dart';
```
Comment (natebosch) [inline pkgs/checks/test/extensions/core_test.dart:1]: [nit] copyright

**Review by natebosch — APPROVED** [review]

File: pkgs/checks/test/extensions/core_test.dart (hunk 2 above)
Comment (kevmoo) [reply pkgs/checks/test/extensions/core_test.dart:1]: Uh...you don't have them anywhere else in this package. Follow-up with a one-off to add em?

**Pull request merged**